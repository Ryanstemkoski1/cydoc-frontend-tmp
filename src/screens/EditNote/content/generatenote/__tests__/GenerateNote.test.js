// import React from 'react';
// import { shallow } from 'enzyme';
// import GenerateNote from '../GenerateNote';
// import configureMockStore from 'redux-mock-store';
// import { initialPhysicalExamState } from '@redux/reducers/physicalExamReducer';
// import { initialReviewOfSystemsState } from 'constants/reviewOfSystemsInitial';

// const mockStore = configureMockStore();

// const initialStore = {
//     reviewOfSystems: initialReviewOfSystemsState,
//     physicalExam: initialPhysicalExamState,
//     medicalHistory: {},
//     surgicalHistory: {},
//     medications: {},
//     allergies: {},
//     socialHistory: {},
//     familyHistory: {},
//     plan: {},
// };

// const store = mockStore(initialStore);

// describe('Generate Note', () => {
//     it('matches snapshot', () => {
//         const component = shallow(<GenerateNote store={store} />);
//         expect(component).toMatchSnapshot();
//     });
// });
