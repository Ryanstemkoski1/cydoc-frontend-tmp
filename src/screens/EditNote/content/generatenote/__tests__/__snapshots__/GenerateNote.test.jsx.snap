// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Generate Note > matches snapshot 1`] = `ShallowWrapper {}`;
